Cubical/Functions/Embedding.agda

```diff
@@ -163,6 +146,23 @@ isEmbedding≡hasPropFibers
            (λ _ → hasPropFibersIsProp _ _)
            (λ _ → isPropIsEmbedding _ _))
 
+-- We use the characterization as hasPropFibers to show that naive injectivity
+-- implies isEmbedding as long as B is an h-set.
+module _
+  {f : A → B}
+  (iSB : isSet B)
```

Comment:
A more descriptive name like "isSetB" could help to make the proof below more readable.

Reply:
Agreed.